internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -609,6 +609,24 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"oidc_issuer_profile": {
+				Type:     pluginsdk.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+						},
+						"issuer_url": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
```

Comment:
We're moving away from this type of schema in the Kubernetes resource where blocks are turned on and off by an `enabled` field since that isn't predominantly how blocks behave throughout the provider.

Could we instead expose this as two separate properties 
```suggestion
			"oidc_issuer_enabled": {
				Type:     pluginsdk.TypeBool,
				Optional: true,
			},
			"oidc_issuer_url": {
				Type:     pluginsdk.TypeString,
				Computed: true,
			},
```